pages/api/v1/posts/on-chain-post.ts

```diff
@@ -590,17 +591,23 @@ export async function getOnChainPost(params: IGetOnChainPostParams) : Promise<IA
 		const preimage = postData?.preimage;
 		const proposalArguments = postData?.proposalArguments  || postData?.callData;
 		const proposedCall = preimage?.proposedCall;
+		let remark = '';
 		let requested: any;
-		if (proposedCall?.args?.amount) {
-			requested = proposedCall.args.amount;
-		} else {
-			if (proposedCall?.args?.calls) {
+		if (proposedCall?.args) {
+			proposedCall.args = convertAnyHexToASCII(proposedCall.args, network);
+			if (proposedCall.args.amount) {
+				requested = proposedCall.args.amount;
+			} else {
 				const calls = proposedCall.args.calls;
 				if (calls && Array.isArray(calls) && calls.length > 0) {
 					const requestedCall = calls.find((call) => !!call.amount);
 					if (requestedCall) {
 						requested = requestedCall.amount;
 					}
+					const remarkCall = calls.find((call) => !!call.remark);
```

Comment:
Small question, Is it possible calls can have multiple remarks? Like in this image, the call has an array with dest and message, can there be a remark over here for each call?
![image](https://github.com/polkassembly/polkassembly/assets/65517966/8c090b15-ba23-4bc2-9fee-ba53c2b2a8df)